scipy/signal/tests/test_signaltools.py

```diff
@@ -1121,23 +1259,25 @@ def test_invalid_dtypes(self, dtype):
         with pytest.raises(ValueError, match="not supported"):
             signal.medfilt2d(in_typed)
 
-    def test_none(self):
+    @skip_xp_backends(np_only=True, reason="object arrays")
+    def test_none(self, xp):
         # gh-1651, trac #1124. Ensure this does not segfault.
-        msg = "dtype=object is not supported by medfilt"
-        with assert_raises(ValueError, match=msg):
+        with assert_raises((ValueError, TypeError)):
             signal.medfilt(None)
 
-    def test_odd_strides(self):
+    @skip_xp_backends(np_only=True, reason="strides are only writeable in NumPy")
+    def test_odd_strides(self, xp):
         # Avoid a regression with possible contiguous
         # numpy arrays that have odd strides. The stride value below gets
         # us into wrong memory if used (but it does not need to be used)
-        dummy = np.arange(10, dtype=np.float64)
+        dummy = xp.arange(10, dtype=xp.float64)
         a = dummy[5:6]
         a.strides = 16
-        assert_(signal.medfilt(a, 1) == 5.)
+        xp_assert_close(signal.medfilt(a, 1),  xp.asarray([5.]))
 
+    @skip_xp_backends(np_only=True, reason="XXX skip for now")
```

Comment:
How come?

Reply:
OK, fixed. Still need to skip on jax because the test chunks the input, feeds the chunks to a ThreadPoolExecutor and collects the results. The last part requires subarray assignment and jax arrays are immutable.